### ISSUE TITLE
Strange units and values for `tage_star`

### ISSUE THREAD

**Issue by MatthieuSchaller** [ICRAR/VELOCIraptor-STF#9]
(copied over from the public github repo)

There seems to be something not quite correct with the field `tage_star` in the output catalogues.

The data is in the range `10^8` - `10^9` but the units are reported as being internal time units, which would typically be something like 9*10^9 years already. 

I don't know whether the issue is in the calculation of the values or whether the units displayed are incorrect.

Also, just to be sure, how are the ages defined? Is there any weighting of any kind for instance?

**Comment by rtobar** [comment]
I believe this is the problem that was fixed via #21. @MatthieuSchaller can you confirm this is the case?

**Comment by MatthieuSchaller** [comment]
So, it's somewhat fixed but a more elegant solution requires more significant changes.

If I set 
```
Stellar_age_to_yr=9.778134136e11
Stellar_age_input_unit_conversion_to_output_unit=1.022690e-12   # year in units of Mpc / (km / s)
```
in the config file and use the latest version (#21) then I get ages which are expressed in the correct units in the snapshots.

However, the more fundamental issue of the unit system being internally inconsistent is still there.

The workaround above forces the ages units to be consistent with the rest (i.e. unit_time == unit_length / unit_velocity for the choice of units we use) but that will not work for a generic set of units without user intervention.
